docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc

```diff
@@ -5972,6 +5972,66 @@ Where :
 * *wsPwd* is the password that is associated to the wsUser. This is configured in the 'Web Services' tab of the switch.
 * *controllerIp* is the IP address of your Unifi controller. This is configured in the 'Definition' tab of the switch.
 
+VLAN Enforcement
+^^^^^^^^^^^^^^^^
+
+In order to configure VLAN enforcement on the Unifi controller, you need first to configure a RADIUS profile, then next a secure Wireless network
+
+Important : You cannot re-use a VLAN ID for dynamic VLAN if it is set as a static value for another SSID on the same AP. So, if I have a SSID set to use VLAN 10, I cannot use VLAN ID 10 for RADIUS controlled VLAN users as those users will not get an IP.
+
+Note that vlan enforcement for an open ssid is not supported yet by the controller and access-point.
+
+AAA Configuration
++++++++++++++++++
+
+image::docs/images/unifi-radius.png[scaledwidth="100%",alt="Unifi Radius Profile"]
+
+Secure SSID
++++++++++++
+
+There is a particularity when you want to be able to deauthenticate a device when this one is connected on the secure SSID.
+
+By default pmksa caching is enable and it occur than even if you send a deauthentication order to the controller, you will not have any new radius request.
```

Comment:
By default pmksa caching is enabled and applies even when a deauthentication request is sent to the controller, meaning you will not get a new RADIUS request afterwards